runtime/parachains/src/configuration.rs

```diff
@@ -114,24 +114,24 @@ pub struct HostConfiguration<BlockNumber> {
 	pub hrmp_max_message_num_per_candidate: u32,
 }
 
-pub trait Trait: frame_system::Trait { }
+pub trait Config: frame_system::Config { }
 
 decl_storage! {
-	trait Store for Module<T: Trait> as Configuration {
+	trait Store for Module<T: Config> as Configuration {
 		/// The active configuration for the current session.
-		Config get(fn config) config(): HostConfiguration<T::BlockNumber>;
+		HostConfig get(fn config) config(): HostConfiguration<T::BlockNumber>;
```

Comment:
note for reviewer: this conflicted thus I changed to HostConfig

Reply:
I guess this pallet is used in some test chains, thus it needs to be migrated, I open such migration/rename in an independant PR for ease reviewing. https://github.com/paritytech/polkadot/pull/2036